src/containers/CreateCommunity.js

```diff
@@ -0,0 +1,434 @@
+          dispatch(navigate(`/create/invite`))
+        }
+      })
+    })
+  }
+
+  render () {
+    const { community, errors } = this.props
+
+    const { expanded } = this.state
+
+    return <Modal title='Create your community.'
+      className='create-community-one'
+      subtitle="Let's get started unlocking the creative potential of your community with Hylo"
+      standalone>
+        <ModalInput label='Name' ref='name' onChange={this.set('name')}/>
+        {errors.nameBlank && <p className='help error'>Please fill in this field.</p>}
+        {errors.nameUsed && <p className='help error'>This name is already in use.</p>}
+        <ModalInput label='URL' ref='url' prefix='https://hylo.com/c/' onChange={this.set('slug')}/>
+        {errors.slugBlank && <p className='help error'>Please fill in this field.</p>}
+        {errors.slugInvalid && <p className='help error'>Use lowercase letters, numbers, and hyphens only.</p>}
+        {errors.slugUsed && <p className='help error'>This URL is already in use.</p>}
+        <ModalInput label='Description' ref='description' type='textarea' onChange={this.set('description')}/>
+        {errors.descriptionBlank && <p className='help error'>Please fill in this field.</p>}
+        <ModalInput label='Invitation Code' ref='beta_access_code' onChange={this.set('beta_access_code')}/>
+        {errors.codeInvalid && <p className='help error'>The code may not contain the slash ("/") character.</p>}
+        {errors.codeBlank && <p className='help error'>Please fill in a code.</p>}
+        {errors.codeUsed && <p className='help error'>This code cannot be used; please choose another.</p>}
+        <div className='toggle-section'>
+          <a onClick={() => this.setState({expanded: !expanded})}>
+            More Options
+            <Icon name={expanded ? 'Chevron-Up2' : 'Chevron-Down2'} />
+          </a>
+        </div>
+        {expanded && <ModalSelect label='Community Type' onChange={this.set('category')}>
+          <option value=''>Pick one:</option>
+          {toPairs(categories).map(([value, label]) => <option key={value} value={value}>
+            {label}
+          </option>)}
+        </ModalSelect>}
+        {expanded && <ModalInput label='Location' onChange={this.set('location')}/>}
+        {expanded && <div className='modal-input'>
+          <label>Logo</label>
+          <div className='small-logo' style={{backgroundImage: `url(${community.avatar_url})`}}></div>
+          <a className='button upload' onClick={() => this.attachImage('avatar')}>Upload</a>
+        </div>}
+        <div className='footer'>
+          <a className='button' ref='submit' onClick={this.submit}>Create</a>
+        </div>
+    </Modal>
+  }
+}
+
+@connect(state => ({
+  community: getCurrentCommunity(state),
+  currentUser: get('people.current', state),
+  invitationEditor: get('invitationEditor', state)
+}))
+export class CreateCommunityInvite extends React.Component {
```

Comment:
One thing we've lost by switching over to this page from the old one is the table of invitations that have already been sent. We should figure out how to restore that.